.github/workflows/push-images.yml

```diff
@@ -0,0 +1,28 @@
+name: Push-Images
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Docker Compose
+        uses: KengoTODA/actions-setup-docker-compose@main
+        with:
+          version: '1.29.2'
+          
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          
+      - name: Login to GitHub Container Registry
+        run: echo $CR_PAT | docker login ghcr.io -u USERNAME --password-stdin
+        env:
+          CR_PAT: ${{ secrets.CR_PAT }}
+      
+      - name: Build and Push Images
+        run: docker-compose build 
+        working-directory: ../../mlflow
```

Comment:
yes, we can use this instead of vanilla docker-compose. 

One thing to note is that the new Dockerfile in #6731 now pip installs from PyPi. This may cause the workflow fail if the MLFlow tag has not been pushed to PyPI at the trigger point (Published released).

Reply:
We publish a release on GitHub after uploading a new version on PyPI.